<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Character view</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <widget class="chardisplay" name="displayChar" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>42</y>
          <width>469</width>
          <height>591</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
          <property name="title">
           <string>Colors</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="frameShadow">
                <enum>QFrame::Plain</enum>
               </property>
               <property name="text">
                <string>Foreground</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="palettebutton" name="pushButton_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="statusTip">
                <string>Click or use the mouse wheel to change color</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Background</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="palettebutton" name="pushButton_3">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="statusTip">
                <string>Click or use the mouse wheel to change color</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Character index</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_2">
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
            <property name="statusTip">
             <string>Char index  - Right click for character map</string>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Character view scale</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox">
            <property name="statusTip">
             <string>Scale of the char view</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Rom font</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="statusTip">
           <string>Not implemented (yet)</string>
          </property>
          <property name="text">
           <string>Multicolor</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1002</width>
     <height>42</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen_rom_font">
     <property name="title">
      <string>Open rom font</string>
     </property>
     <addaction name="actionDefault"/>
     <addaction name="actionUpper_Lower_case"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="menuOpen_rom_font"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionShift_Right"/>
    <addaction name="actionShift_Left"/>
    <addaction name="actionShift_Up"/>
    <addaction name="actionShift_Down"/>
    <addaction name="separator"/>
    <addaction name="actionMirror_Horizontaly"/>
    <addaction name="actionMirror_Verticaly"/>
    <addaction name="separator"/>
    <addaction name="actionRotate_CW"/>
    <addaction name="actionRotate_CCW"/>
    <addaction name="separator"/>
    <addaction name="actionInvert"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionShift_Right"/>
   <addaction name="actionShift_Left"/>
   <addaction name="actionShift_Up"/>
   <addaction name="actionShift_Down"/>
   <addaction name="actionMirror_Horizontaly"/>
   <addaction name="actionMirror_Verticaly"/>
   <addaction name="actionRotate_CW"/>
   <addaction name="actionRotate_CCW"/>
   <addaction name="actionInvert"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="statusTip">
    <string>Open a character set from disk</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="statusTip">
    <string>Quit the application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="statusTip">
    <string>Save the character set to disk</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/cut.png</normaloff>:/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="statusTip">
    <string>Copy character data and clear</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/copy.png</normaloff>:/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="statusTip">
    <string>Copy character data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/paste.png</normaloff>:/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="statusTip">
    <string>Paste character data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionShift_Right">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/shift-right.png</normaloff>:/images/shift-right.png</iconset>
   </property>
   <property name="text">
    <string>Shift Right</string>
   </property>
   <property name="statusTip">
    <string>Shift character to the right</string>
   </property>
  </action>
  <action name="actionShift_Left">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/shift-left.png</normaloff>:/images/shift-left.png</iconset>
   </property>
   <property name="text">
    <string>Shift Left</string>
   </property>
   <property name="statusTip">
    <string>Shift character to the left</string>
   </property>
  </action>
  <action name="actionShift_Up">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/shift-up.png</normaloff>:/images/shift-up.png</iconset>
   </property>
   <property name="text">
    <string>Shift Up</string>
   </property>
   <property name="statusTip">
    <string>Shift character upwards</string>
   </property>
  </action>
  <action name="actionShift_Down">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/shift-down.png</normaloff>:/images/shift-down.png</iconset>
   </property>
   <property name="text">
    <string>Shift Down</string>
   </property>
   <property name="statusTip">
    <string>Shift character downwards</string>
   </property>
  </action>
  <action name="actionMirror_Horizontaly">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/mirror-horiz.png</normaloff>:/images/mirror-horiz.png</iconset>
   </property>
   <property name="text">
    <string>Mirror Horizontaly</string>
   </property>
   <property name="statusTip">
    <string>Mirror character horizontaly</string>
   </property>
  </action>
  <action name="actionMirror_Verticaly">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/mirror-vert.png</normaloff>:/images/mirror-vert.png</iconset>
   </property>
   <property name="text">
    <string>Mirror Verticaly</string>
   </property>
   <property name="statusTip">
    <string>Mirror character verticaly</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionRotate_CW">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/rotatecw.png</normaloff>:/images/rotatecw.png</iconset>
   </property>
   <property name="text">
    <string>Rotate CW</string>
   </property>
   <property name="statusTip">
    <string>Rotate character clockwise</string>
   </property>
  </action>
  <action name="actionRotate_CCW">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/rotateccw.png</normaloff>:/images/rotateccw.png</iconset>
   </property>
   <property name="text">
    <string>Rotate CCW</string>
   </property>
   <property name="statusTip">
    <string>Rotate character counter-clockwise</string>
   </property>
  </action>
  <action name="actionDefault">
   <property name="text">
    <string>Default</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionUpper_Lower_case">
   <property name="text">
    <string>Upper/Lower case</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="statusTip">
    <string>Start a new character set from scratch</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionInvert">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/invert.png</normaloff>:/images/invert.png</iconset>
   </property>
   <property name="text">
    <string>Invert</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Char64...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>chardisplay</class>
   <extends>QWidget</extends>
   <header>chardisplay.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>palettebutton</class>
   <extends>QPushButton</extends>
   <header>palettebutton.h</header>
   <slots>
    <signal>currentIndexChanged(int)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
